docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,330 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hcls-cluster
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hcls-01
+  region: us-central1
+  zone: us-central1-c
+
+  network_name: hcls-cluster-net
+  subnetwork_name: primary-subnet
+  bucket_name_software: user-provided-software
+  bucket_name_inputs: ## Set Inputs Bucket Name Here ##
+  bucket_name_outputs: ## Set Outputs Bucket Name Here ##
+  use_deployment_name_in_bucket_name: false
+  homefs_server_ip:  ## Set Home Filestore IP Here ##
+  appsfs_server_ip:  ## Set Apps Filestore IP Here ##
+
+deployment_groups:
+- group: enable_apis
+  modules:
+
+  ### Enable APIs ###
+
+  - id: services-api
+    source: community/modules/project/service-enablement
+    settings:
+      gcp_service_list:
+      - file.googleapis.com
+      - iam.googleapis.com
+      - pubsub.googleapis.com
+      - secretmanager.googleapis.com
+      - serviceusage.googleapis.com
+      - compute.googleapis.com
+      - stackdriver.googleapis.com
+
+- group: setup
+  modules:
+
+  ### Network ###
+
+  - id: network-create
+    source: modules/network/vpc
+
+  ### Resource Monitoring ###
+
+  - id: hpc-dash
+    source: modules/monitoring/dashboard
+
+  ### Storage ###
+
+  - id: homefs-create
+    source: modules/file-system/filestore
+    use: [network-create]
+    settings: {filestore_share_name: home}
+
+  - id: appsfs-create
+    source: modules/file-system/filestore
+    use: [network-create]
+    settings: {filestore_share_name: apps}
+
+  - id: bucket-software-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_software)
+
+  - id: bucket-input-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_inputs)
+
+  - id: bucket-output-create
```

Comment:
@cboneti, we are leaning towards removing the outputs bucket. The demo doesn't use it and it seems likely that an output bucket would be something generated outside of a blueprint on a 1 per user basis with specific permissions.
Thoughts?

Reply:
funny, I had that thinking for the input, but not for the output. See, the input source (bucket) is typically something you don't change as much, I would say you even do several simulations on the same exact input (with different parameters), whereas you want outputs to be passed on. The reality is that it is likely that both input and output buckets will be reused a lot. 

To simplify, why not create a single bucket with inputs and outputs in it?

Reply:
I think there are different use cases. There can be a read only inputs library that does not change. I think this makes sense to have as a bucket so it is easily visible and jobs don't have to rely on the public internet.

Then I think there is a valid use case for the output bucket as well (as you describe). I think the concern here is that the outputs bucket would probably not be a shared resource. A scientist would be cautious about writing data somewhere that could be overwritten by anyone else using the cluster. 

My proposal is to leave them both in for now. Write a doc section about this and get consensus before modifying further. I think I am in the camp of it is useful to have the output bucket to show the pattern even if in real life their might be a bucket per user pattern that is not captured in the blueprint.